packages/reactive-element/src/reactive-element.ts

```diff
@@ -1082,7 +1082,9 @@ export abstract class ReactiveElement
   // Note, this is an override point for polyfill-support.
   // @internal
   _$didUpdate(changedProperties: PropertyValues) {
-    this.__controllers?.forEach((c) => c.hostUpdated?.());
+    this.__controllers?.forEach((c) => {
+      c.hostUpdated?.();
```

Comment:
I'm not sure Terser will undo this change, since it can't know that hostUpdate() returns void. I'd rather keep the shorter form.

Reply:
What would you say to making this a for-of?

Reply:
Can you check the size impact? Both forms work, I prefer the smaller.